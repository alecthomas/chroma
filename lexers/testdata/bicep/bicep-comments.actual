// This is your primary NIC.
resource nic1 'Microsoft.Network/networkInterfaces@2020-06-01' = {
}

/*
  This Bicep file assumes the key vault already exists and
  is in same subscription and resource group as the deployment.
*/
param existingKeyVaultName string