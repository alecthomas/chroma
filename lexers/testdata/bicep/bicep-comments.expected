[
  {"type":"CommentSingle","value":"// This is your primary NIC."},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"KeywordDeclaration","value":"resource"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"nic1"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"'Microsoft.Network/networkInterfaces@2020-06-01'"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":"}"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"Operator","value":"/*"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"Text","value":"This"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"Bicep"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"file"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"assumes"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"the"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"key"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"vault"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"already"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"exists"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"and"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"Text","value":"is"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Keyword","value":"in"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"same"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameFunction","value":"subscription"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"and"},
  {"type":"TextWhitespace","value":" "},
  {"type":"KeywordDeclaration","value":"resource"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"group"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"as"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"the"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameFunction","value":"deployment"},
  {"type":"Punctuation","value":"."},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Operator","value":"*/"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"KeywordDeclaration","value":"param"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"existingKeyVaultName"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameFunction","value":"string"}
]
